site/features.md

```diff
@@ -248,6 +249,29 @@ invocation:
     host1$ weave detach net:default net:10.2.2.0/24 net:10.2.3.0/24 $C
     10.2.1.3 10.2.2.3 10.2.3.1
 
+### <a name="restarting-containers"></a>Restarting containers
+
+If you restart a container, it will retain the same IP addresses on
+the weave network:
+
+(assuming `DOCKER_HOST` is pointing at the weave Docker API proxy)
+
+    host1$ docker run --name a1 -tdi ubuntu
+    f76b09a9fcfee04551dbb8d951d9a83e7e7d55126b02fd9f44f9f8a5f07d7c96
+    host1$ weave ps a1
+    a1 1e:dc:2a:db:ef:ff 10.32.0.3/12
+    host1$ docker restart a1
+    host1$ weave ps a1
+    a1 16:c0:6f:5d:c5:73 10.32.0.3/12
+
+There is also a `weave restart` command, if you are not using the
+Docker API proxy:
+
+    host1$ weave restart b1
+
+Note that if Docker restarts a container under a restart policy it
+will not be re-attached to the weave network.
+
```

Comment:
The statement isn't restricted to those using IPAM; as it happens restart via the proxy worked with manually-assigned addresses already but we didn't document it.